scenarios/microsoft-azure-confidential-compute-v1/configuration.metadata.json

```diff
@@ -1,164 +1,164 @@
+                        "Confidential virtual machine",
+                        "Ubuntu Server 20.04 LTS (Confidential VM preview) - Gen 2 image",
+                        "Standard_DC2as_v5 - 2 vcpu, 8 GiB memory",
+                        "Confidential compute encryption disabled"
+                    ]
+                },
+                {
+                    "pivotName": "disks",
+                    "content": [
+                        "32 GiB / Premium SSD LRS",
+                        "No disk encryption"
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "ProdVM",
+            "title": "Confidential production ready virtual machine",
+            "subTitle": "Best for production and active protection against advanced and persistent attack techniques.",
+            "cost": {
+                "value": "40",
+                "score": 2,
+                "currency": "USD"
+            },
+            "maintenanceScore": 2,
+            "pivotValues": [
+                {
+                    "pivotName": "virtualMachine",
+                    "content": [
+                        "Confidential virtual machine",
+                        "Ubuntu Server 20.04 LTS (Confidential VM preview) Gen 2 image",
+                        "Standard_DC4ads_v5 - 4 vcpus, 16 GiB memory",
+                        "Confidential compute encryption enabled"
+                    ]
+                },
+                {
+                    "pivotName": "disks",
+                    "content": [
+                        "32 GiB / Premium SSD LRS",
+                        "Platform-managed key"
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "ProdVMSS",
+            "title": "Confidential production ready VMSS",
+            "subTitle": "Best for production and active protection against advanced and persistent attack techniques.",
+            "cost": {
+                "value": "40",
+                "score": 3,
+                "currency": "USD"
+            },
+            "maintenanceScore": 3,
+            "pivotValues": [
+                {
+                    "pivotName": "virtualMachine",
+                    "content": [
+                        "Confidential virtual machines within a flexible VM scale set for maximum scaling",
+                        "Ubuntu Server 20.04 LTS (Confidential VM preview) - Gen 2 image",
+                        "Standard_DC4ads_v5 - 4 vcpus, 16 GiB memory",
+                        "Confidential compute encryption unavailable"
+                    ]
+                },
+                {
+                    "pivotName": "virtualMachineScaleSet",
+                    "content": [
+                        "Flexible scale set used for load balancing against the virtual machine instances",
+                        "Autoscaler enabled with a minimum instance count of 2"
+                    ]

```

Comment:
Why has this file changed ? Can you rebase your branch onto the latest staging ?